_posts/2016-07-26-webhooks.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Introducing webhooks
+author: Thomas Hipp
+layout: blogpost
+---
+
+Hello!
+
+I am pleased to announce one of the new features which will be shipped with
+Portus version 2.1.
+
+#### What are webhooks and how do they work?
+
+Simply put, a webhook is a user-defined HTTP callback which notifies a
+specified endpoint about some sort of change.
+In Portus, this change can be either a push or delete event, i.e. webhooks will
+be triggered when you push or delete an image.
+
+Creating a webhook in Portus is as simple as defining a URL.
+Although every HTTP request also requires a request method, this does not
+explicitly need to be set by the user, and defaults to `GET` in Portus.
+Another option, or better header, which has a default value is `content-type`.
+It defaults to `application/json` but can be changed if needed.
+Other customizable options include  username and password, and custom HTTP
+headers.
+Per default, these are left empty if not explicitly specified.
+Furthermore, each webhook can be temporary disabled if it's not needed, and be
+enabled again any time.
+Note: a newly created webhook is disabled and needs to be enabled before it can
```

Comment:
Maybe you can embed this in a block like this:

```HTML
<div class="alert alert-info">
  <strong>Note</strong>: a newly created webhook is disabled and needs to be enabled before it can be used.
</div>
```